### PR TITLE
[CSL-2699] Add secure and samesite

### PR DESCRIPTION
**Pull request by esezen**

### CODE REVIEW

File: src/constructorio-id.js [hunk 1]
```diff
@@ -61,6 +63,12 @@
       if (this.cookie_domain) {
         cookie_data += '; domain=' + this.cookie_domain;
       }
+      if (this.cookie_secure) {
+        cookie_data += '; secure';
+      }
+      if (this.cookie_samesite) {
+        cookie_data += '; samesite=' + this.cookie_samesite;
```
Comment (mocca102) [inline src/constructorio-id.js:70]: This won't affect functionality but it seems it's a convention to write it as capitalized as `SameSite`

File: src/constructorio-id.js [hunk 2]
```diff
@@ -61,6 +63,12 @@
       if (this.cookie_domain) {
         cookie_data += '; domain=' + this.cookie_domain;
       }
+      if (this.cookie_secure) {
+        cookie_data += '; secure';
```
Comment (mocca102) [inline src/constructorio-id.js:67]: This won't affect functionality but it seems it's a convention to write it as capitalized as `Secure`

**Review by mocca102 — APPROVED** [review]
LGTM! Just left two minor comments

File: src/constructorio-id.js (hunk 1 above)
Comment (esezen) [reply src/constructorio-id.js:70]: Correct but that applies to all params (domain, path, expires) and we don't do that for the rest of the params, I was planning to keep it consistent with the rest of the file

Comment (mocca102) [reply src/constructorio-id.js:70]: Then we're good 👍

**Review by jjl014 — APPROVED** [review]
LGTM! 🚀

**Pull request merged**